internal/attestation/crafter/crafter_test.go

```diff
@@ -496,15 +496,17 @@ func (s *crafterSuite) TestAddMaterialsAutomatic() {
 
 			backend := &casclient.CASBackend{Uploader: uploader}
 
-			if tc.exceedSize {
+			// Establishing a maximum size for the artifact to be uploaded to the CAS causes an error
+			// if the value is exceeded
+			if tc.wantErr {
```

Comment:
this is fine for now but whenever we add a string validation this test will not work I think. Either way, that's smth for the future :)